NOTES.UNIX

```diff
@@ -28,3 +28,12 @@
 
     $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
         -Wl,-rpath,'$(LIBRPATH)'
+
+ On most GNU systems, a better choice is to use the new dtags, like this:
+
+    $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
+        -Wl,--enable-new-dtags,-rpath,'$(LIBRPATH)'
+
+ This sets DT_RUNPATH instead of DT_RPATH.  DT_RUNPATH is considered after
+ the environment variable LD_LIBRARY_PATH, while DT_RPATH is considered
+ before that environment variable (making it useless).
```

Comment:
It isn't "useless" - it is just different to what some people want. There are places for both behaviours.

Reply:
Ah, I need to be more precise.  "it" refers to the environment variable `LD_LIBRARY_PATH`, not to what different people might desire.